src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionService.kt

```diff
@@ -225,10 +229,33 @@ class LicenceConditionService(
       additionalConditionData = request.data.transformToEntityAdditionalData(additionalCondition),
       expandedConditionText = request.expandedConditionText,
     )
+    checkFormattedText(updatedAdditionalCondition)
     additionalConditionRepository.saveAndFlush(updatedAdditionalCondition)
 
     val username = SecurityContextHolder.getContext().authentication.name
     val updatedLicence = licenceEntity.copy(dateLastUpdated = LocalDateTime.now(), updatedByUsername = username)
     licenceRepository.saveAndFlush(updatedLicence)
   }
+
+  fun checkFormattedText(additionalCondition: EntityAdditionalCondition) {
```

Comment:
This is temporary just to ensure that we're correctly formatting conditions. 

Once we can confirm this then we will use the serverside generated text and remove the frontend code